README.md

```diff
@@ -1195,6 +1195,7 @@ Update Time, five active automations, webhooks.
   * [MDB GO](https://mdbgo.com/) - Free hosting for one project with two weeks Container TTL, 500 MB RAM per project, SFTP - 1G disk space.
   * [Patr Cloud](https://patr.cloud/) — An easy-to-use cloud platform, among its paid services it offers to host three static sites for free.
   * [Serv00.com](https://serv00.com/) — 3 GB of free web hosting with daily backups (7 days). Support: Crontab jobs, SSH access, repositories (GIT, SVN, and Mercurial), support: MySQL, PostgreSQL, MongoDB, PHP, Node.js, Python, Ruby, Java, Perl, TCL/TK, Lua, Erlang, Rust, Pascal, C, C++, D, R, and many more.
+  * [Zeabur](zeabur.com) - Deploy instantly and scale infinitely. Free for hosting serverless functions and static sites, free SSL and sub domain.
```

Comment:
It says $0+resource usage.  So that doesnt sound to me like its a free tier.

Reply:
For serverless functions it's free to use

Reply:
It might be, but thats not what the site says in clear terms.